CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ## 1.20.0 (Not yet released)
 
 * Updated the auto-generated launch.json to use new mechanism for starting web browser. For more information: https://aka.ms/VSCode-CS-LaunchJson-WebBrowser
+* Fadeout support [#2873](https://github.com/OmniSharp/omnisharp-vscode/pull/2873)
```

Comment:
"Added support for fading unnecessary code and using statements"

Reply:
Updated